shardingsphere-features/shardingsphere-sharding/shardingsphere-sharding-distsql/shardingsphere-sharding-distsql-handler/src/main/java/org/apache/shardingsphere/sharding/distsql/handler/update/DropDefaultStrategyStatementUpdater.java

```diff
@@ -57,6 +63,15 @@ private void checkCurrentRuleConfiguration(final String schemaName, final Shardi
         DistSQLException.predictionThrow(currentRuleConfig != null, new RequiredRuleMissedException("Sharding", schemaName));
     }
     
+    @Override
+    public boolean hasAnyOneToBeDropped(final DropDefaultShardingStrategyStatement sqlStatement, final ShardingRuleConfiguration currentRuleConfig) {
+        if (sqlStatement.getDefaultType().equalsIgnoreCase(ShardingStrategyLevelType.TABLE.name())) {
+            return null != currentRuleConfig && null != currentRuleConfig.getDefaultTableShardingStrategy();
+        } else {
+            return null != currentRuleConfig && null != currentRuleConfig.getDefaultDatabaseShardingStrategy();
```

Comment:
I'm wondering compare 「there is no `else` keyword in this method」 with this way, which is better.
if somebody review on this, please give me some tips.

Reply:
According to [Code of Conduct](https://shardingsphere.apache.org/community/en/contribute/code-conduct/) :
"Use guard clauses in priority."
I think it would be better not to have the else here.